qi-doc/scribblings/intro.scrbl

```diff
@@ -69,6 +69,29 @@ Qi is a hosted language on the @hyperlink["https://racket-lang.org/"]{Racket pla
 
  Additionally, Qi itself uses few and carefully benchmarked dependencies, so that the load-time overhead of @racket[(require qi)] is minimal.
 
+@subsection{About Qi's Release Practices}
+
+Qi follows the @hyperlink["http://timothyfitz.com/2009/02/10/continuous-deployment-at-imvu-doing-the-impossible-fifty-times-a-day/"]{continuous deployment} model of development. This means that fresh changes are immediately pushed to the main branch of development after they pass a rigorous and comprehensive suite of tests.
+
+Furthermore, Qi packages on the Racket package index point to this main branch on the source host, so that running @racket[raco pkg install qi] or @racket[raco pkg update qi] will always get the version on the @racket[main] branch, reflecting the latest improvements.
+
+This doesn't mean that you must use this version, however. The expressiveness of modern version control systems allows us to define diverse versioning protocols directly on the versioning backend to best support diverse usage needs. Towards this end, Qi follows these conventions:
```

Comment:
What does "on the versioning backend" mean? I've never heard of such a thing, and now I'm curious!

Reply:
Oh, just Git! Reflecting on my unwitting use of a nonstandard term, I think it reflects the way I'd like to think about Git in relation to package management systems -- that all versioning should be left to the versioning "backend" whose sole purpose is to manage versions, and we should not maintain an independent notion of versions "on the frontend." (e.g. in Raco / the Racket package index).

Reply:
Appreciate the new clarification! It's helpful to tie new terms to old ones ;)